WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletStatsViewModel.cs

```diff
@@ -66,5 +74,14 @@ private void UpdateProps()
 
 		LargestExternalKeyGap = _wallet.KeyManager.CountConsecutiveUnusedKeys(isInternal: false, ignoreTail: true);
 		LargestInternalKeyGap = _wallet.KeyManager.CountConsecutiveUnusedKeys(isInternal: true, ignoreTail: true);
+
+		var singleCoinjoins = _walletViewModel.History.Transactions.OfType<CoinJoinHistoryItemViewModel>().ToList();
+		var groupedCoinjoins = _walletViewModel.History.Transactions.OfType<CoinJoinsHistoryItemViewModel>().ToList();
+		var nestedCoinjoins = groupedCoinjoins.SelectMany(x => x.Children).ToList();
+		var nonCoinjoins = _walletViewModel.History.Transactions.Where(x => !x.IsCoinJoin).ToList();
+
+		TotalTransactionCount = singleCoinjoins.Count + groupedCoinjoins.Count + nestedCoinjoins.Count + nonCoinjoins.Count;
+		NonCoinjointransactionCount = nonCoinjoins.Count;
+		CoinjoinTransactionCount = singleCoinjoins.Count + groupedCoinjoins.Count + nestedCoinjoins.Count;
```

Comment:
The grouped coinjoins should not be counted.

```suggestion
		TotalTransactionCount = singleCoinjoins.Count + nestedCoinjoins.Count + nonCoinjoins.Count;
		NonCoinjointransactionCount = nonCoinjoins.Count;
		CoinjoinTransactionCount = singleCoinjoins.Count + nestedCoinjoins.Count;
```